frontend/src/Commons/Forms/FormModule/FormModule.style.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @desc Styling with BC Design Tokens and Emotion for Form Module
+ * @author @GDamaso
+ */
+import styled from '@emotion/styled';
+import * as tokens from '@bcgov/design-tokens/js';
+import screenSizes from '@Constants/ScreenSize';
+
+const StyledFormContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+  width: 95%;
+
+  @media (min-width: ${screenSizes.desktop}) {
+    max-width: 70%;
+  }
+`;
+
+const StyledFormContent = styled.div`
+  width: 100%;
+  color: ${tokens.typographyColorPrimary};
+  border-radius: 2px;
+  border-top-left-radius: 1px;
+  border-top-right-radius: 1px;
+  border: 1px solid ${tokens.themeGray40};
+  padding: 10px;
+  padding-bottom: 15px;
+  padding-top: 0px;
```

Comment:
Things like this can just be reduced to 
`padding: 0 10px 15px 10px;`

You're already specifying 3/4 sides for the padding to occur